### PR TITLE
Dockerise reporting

### PR DESCRIPTION
**Pull request by Zhi-704**
Added dockerfile to notifications, changed folder name, added lambda handling

### CODE REVIEW

File: reporting/requirements.txt
Comment (Lasped13) [inline reporting/requirements.txt:1]: Why was this deleted?

File: notfications/requirements.txt
Comment (Lasped13) [inline notfications/requirements.txt:1]: Interesting choice but fair enough.

File: notfications/dockerfile
Comment (Lasped13) [inline notfications/dockerfile:1]: Looks good. Well done!

File: dashboard/database.py
Comment (Lasped13) [inline dashboard/database.py:1]: Why are the old sql queries back? They are less inefficient.

File: notfications/report.py [hunk 1]
```diff
@@ -125,8 +125,8 @@ def publish_list_to_topic(sns_client: client,
         item_type = item['type'].title()
         item_url = item['url']
 
-        message += f"{item_type} '{item_name}' by '{
-            item_author}' is currently trending!\n"
+        message += f'''{item_type} '{item_name}' by '{
+            item_author}' is currently trending!\n'''
```
Comment (Lasped13) [inline notfications/report.py:129]: " > ' imo

File: notfications/report.py [hunk 2]
```diff
@@ -30,8 +30,8 @@ def get_sns_client() -> client:
     '''Creates sns client'''
     return client('sns',
                   aws_access_key_id=ENV["ACCESS_KEY"],
-                  aws_secret_access_key=ENV["SECRET_ACCESS_KEY"]
-                  )
+                  aws_secret_access_key=ENV["SECRET_ACCESS_KEY"],
+                  region_name='eu-west-2')
```
[the remark this answers is not in the document]
Comment (Zhi-704) [reply notfications/report.py:34]: I don't see much reason to turn this into an env as all our AWS resources show eu-west-2

File: notfications/report.py (hunk 1 above)
Comment (Zhi-704) [reply notfications/report.py:129]: The lambda image doesn't recognise " for multiple lines which is why I changed it to ' instead.

**Review by Lasped13 — APPROVED** [review]

File: reporting/requirements.txt
Comment (Zhi-704) [reply reporting/requirements.txt:1]: Reporting folder doesn't exist anymore

File: dashboard/database.py
Comment (Zhi-704) [reply dashboard/database.py:1]: Yeah not sure why they returned when I merged with main

**Comment by Zhi-704** [comment]
Cllose #126
